.github/workflows/build-test.yml

```diff
@@ -179,16 +179,8 @@ jobs:
           skip-label: 'skip telemetry'
           failure-message: 'TELEMETRY.md should be updated when adding or removing telemetry. Please run "npm run generateTelemetry"'
 
-    # See issue https://github.com/microsoft/vscode-jupyter/issues/10343. We need a better way to validate. Perhaps nightly.
-    # - name: Validate TELEMETRY.md file
-    #    run: npm run validateTelemetryMD
-
-    #   - name: 'Check for news entry'
-    #     uses: brettcannon/check-for-changed-files@v1.1.0
-    #     with:
-    #       file-pattern: 'news/*/*.md'
-    #       skip-label: 'skip news'
-    #       failure-message: 'News entry file missing; see news/README.md for instructions (the ${skip-label} label can be used to pass this check)'
+      - name: Validate TELEMETRY files
+        run: npm run validateTelemetryFiles
```

Comment:
The src/gdpr.ts file will need to be kept upto date with the latest information.
hence this will have to be updated in every PR if there are any changes to the telemetry.

In the past this was very painful as changes to line number caused the telemetry.md file to change, as we no longer have that stuff in he md file it will not occur as frequently, unless of course someone change anything related to telemetry.